docs/quickstart.rst

```diff
@@ -90,8 +90,9 @@ backed up and that the ``prune`` command is keeping and deleting the correct bac
     export BORG_PASSCOMMAND='pass show backup'
 
     # some helpers and error handling:
-    function info  () { echo -e "\n"`date` $@"\n" >&2; }
-    trap "echo `date` Backup interrupted >&2; exit 2" SIGINT SIGTERM
+    info() { printf "\n%s %s\n\n" "$( date )" "$*" >&2; }
+    # OR: info() { echo; echo "$( date ) $*" >&2; echo; }
+    trap 'echo $( date ) Backup interrupted >&2; exit 2' INT TERM
```

Comment:
Shellcheck said that in POSIX signals are not prefixed with "SIG" and thus I removed it. The [ubuntu man page](http://manpages.ubuntu.com/manpages/trusty/man1/trap.1posix.html) is a bit confusing to me on this topic:
> Implementations may also accept the names with the SIG prefix; no known historical shell does so. 

Don't know what a historical shell is for them. For full compatibility you can only still use the signal numbers directly. (when signint and sigterm are the same on all systems…)